### ISSUE TITLE
Resume the annotation of the images .

### ISSUE THREAD

**Issue by roopeshn28** [CSAILVision/LabelMeAnnotationTool#71]
Hi, I'm using this tool for my clients, they are having issue that when ever they login it points to the first image only and they need to press next untill they reach the image they need to annotate.Can you please tell how do i fix

**Comment by Abhijit-2592** [comment]
@roopeshn28  I don't think you can directly do it. The work around I use is. You can write a small script that finds whether an image is labeled or not by checking if a corresponding xml file exists and populate a new collection accordingly, and point the url to that collection

**Comment by roopeshn28** [comment]
@Abhijit-2592  can you tell me how i can do that?Bcz im new to the backend programming
